src/main/java/MassimoBoi/FileClass.java

```diff
@@ -0,0 +1,62 @@
+package MassimoBoi;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class FileClass {
+
+    public static void loadList(List<Task> taskList) throws FileNotFoundException {
+        File f = new File("./src/main/java/list.txt");
+        Scanner s = new Scanner(f);
+        while(s.hasNext()){
+            Task newTask;
+            String line = s.nextLine();
+
+            String[] lineArray = line.split("\\|");
+            if(line.contains("[T]")){
```

Comment:
```suggestion
            if (line.contains("[T]")) {
```

Reply:
please check this format for other statements, including 'while' statements etc.